asv_bench/benchmarks/inference.py

```diff
@@ -1,77 +1,84 @@
-from .pandas_vb_common import *
-import pandas as pd
+import numpy as np
+import pandas.util.testing as tm
+try:
+    import pandas._libs.lib as lib
+except ImportError:
+    import pandas.lib as lib
```

Comment:
I would keep this in `pandas_vb_common`, you can just import lib from there (as this can then be reused in multiple files)